packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md

```diff
@@ -172,3 +182,52 @@ enrollments.find { it.user().name == "myUser" }?.let { pushEnrollment ->
         .onFailure { println("failure") }
 }
 ```
+
+Alternatively, you can retrieve undelivered challenges using the [MyAccount App Authenticators API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/AppAuthenticator/#tag/AppAuthenticator/operation/listAppAuthenticatorPendingPushNotificationChallenges).
+
+### Access token management
+
+> **Note:** To enable the JWT bearer grant type:
+>  * Send a PUT request to `/apps/{appId}`. Ensure that the `grant_types` array contains the following string:
+>    `urn:ietf:params:oauth:grant-type:jwt-bearer`
+>  * If you use custom authorization servers, update the policy rules to update the grant type.
+
+The SDK communicates with an Okta server using HTTPS protocol and requires an access token for user authentication and authorization. For authentication flows and requesting access tokens, use the latest version of the [Okta OIDC SDK](https://github.com/okta/okta-oidc-android). To enroll a push authenticator, the access token needs the `okta.myAccount.appAuthenticator.manage` scope. You can also use this scope for the following operations:
```

Comment:
should be 
[Okta Mobile Kotlin](https://github.com/okta/okta-mobile-kotlin)

Reply:
Done. Thanks!